docs/synthetic-size.md

```diff
@@ -1,5 +1,41 @@
 # Synthetic size
 
+## How to get the data
+
+Pageserver provides a HTTP API for getting the synthetic size of a tenant
+along with the data that was used to calculate it. Usage examples:
+
+1. This query returns the synthetic size of the tenant, along with the "raw" data
+That is returned in the `segments` and `timeline_inputs` fields.
+
+```
+curl localhost:9898/v1/tenant/5e1de642394b00a0a583a088e8276b98/synthetic_size | jq
```

Comment:
```suggestion
curl localhost:9898/v1/tenant/5e1de642394b00a0a583a088e8276b98/synthetic_size | jq .
```

Didn't test this, but I think jq always wants some argument as to what to extract.